web/satellite/tslint.json

```diff
@@ -12,61 +12,60 @@
   },
   "rules": {
     "function-constructor": true,
-    // Enforces vertical alignment.
-    "align": [true, "parameters", "statements"],
-    // Enforces use of T[] if T is a simple type.
-    "array-type": [true, "array-simple"],
-    // Enforces PascalCased class and interface names.
+    "align": [true, "parameters", "statements"],    
+    "array-type": [true, "array-simple"],  
+    "arrow-return-shorthand": true,
+    "check-format": true,
     "class-name": true,
-    // Enforces formatting rules for single-line comments.
     "comment-format": [true, "check-space"],
-    "quotemark": [true, "single", "avoid-escape"],
-    // Ensures the file ends with a newline.
-    "eofline": true,
-    "indent": [true, "spaces", 4],
-    // Ensures proper spacing between import statement keywords.
-    "import-spacing": true,
-    "interface-name": false,
-    "ordered-imports": false,
-    // Enforces consistent semicolon usage at the end of every statement.
-    "semicolon": [true, "always"],
-    // Enforces braces for if/for/do/while statements.
+    "comment-type": [true, "doc", "singleline"],
     "curly": [true, "ignore-same-line"],
-    // Enforces blank line before return when not the only line in the block.
-    "newline-before-return": true,
-    // Disallows multiple variable definitions in the same declaration statement.(Exception for loops)
-    "one-variable-per-declaration": [true, "ignore-for-loop"],
-    "object-literal-sort-keys": false,
-    "whitespace": [
-      true,
-      "check-branch", // checks branching statements (if/else/for/while) are followed by whitespace.
-      "check-decl",  // checks that variable declarations have whitespace around the equals token.
-      "check-operator", // checks for whitespace around operator tokens.
-      "check-module", // checks for whitespace in import & export statements.
-      "check-separator", // checks for whitespace after separator tokens (,/;).
-      "check-type-operator", // checks for whitespace between type operators | and &.
-      "check-preblock" // checks for whitespace before the opening brace of a block.
-    ],
-    // Recommends to use an early exit instead of a long if block.
     "early-exit": true,
-    // Bans the use of specified console methods.
+    "eofline": true,
+    "indent": [true, "spaces", 4],    
+    "interface-name": false,
+    "import-spacing": true,
+    "no-async-without-await": true,
+    "no-boolean-literal-compare": true,
+    "no-conditional-assignment": true,
+    "no-consecutive-blank-lines": [true, 1],
     "no-console": [true, "log"],
     "no-default-export": false,
```

Comment:
should we disable default export?

Reply:
it seems that vue components should be default exported